radis/lbl/factory.py

```diff
@@ -1554,10 +1554,12 @@ def _get_S0(self, Ia_arr):
         if "S0" in df.columns:
             return df["S0"]
 
+        self.calc_einstein_coefficients()
```

Comment:
This may need some discussion on Slack. I'm worried this function is quite expensive to compute. (deal with other comments before this one :)  )

Reply:
https://radis-radiation.slack.com/archives/C0224856BUM/p1625263911008400 Link to the discussion